<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NansheDictionaryLearningDrawer</class>
 <widget class="QWidget" name="NansheDictionaryLearningDrawer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="Normalize">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="NormalizeLayout">
        <item>
         <widget class="QLabel" name="NormalizeTitle">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How to normalize data. L&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; norm recommended.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Normalize&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Norm_2">
          <item>
           <widget class="QLabel" name="NormLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How to normalize data. L&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; norm recommended.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Norm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="NormValueSelection">
            <item>
             <property name="text">
              <string>inf</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="NansheDictionaryLearningFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="NansheDictionaryLearningTitleLayout">
        <item>
         <widget class="QLabel" name="NansheDictionaryLearningTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Arguments to be used by SPAMS dictionary learning algorithm. ( &lt;a href=&quot;http://spams-devel.gforge.inria.fr/doc-python/html/doc_spams004.html#sec5&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://spams-devel.gforge.inria.fr/doc-python/html/doc_spams004.html#sec5&lt;/span&gt;&lt;/a&gt; )&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dictionary Learning&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NansheDictionaryLearningInfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="NansheDictionaryLearningValuesLayout">
        <item>
         <layout class="QHBoxLayout" name="KLayout">
          <item>
           <widget class="QLabel" name="KLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the dictionary.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="KValue">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Gamma1Layout">
          <item>
           <widget class="QLabel" name="Gamma1Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Gamma1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Gamma1Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Gamma2Layout">
          <item>
           <widget class="QLabel" name="Gamma2Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Gamma2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Gamma2Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="NumThreadsLayout">
          <item>
           <widget class="QLabel" name="NumThreadsLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of threads to use.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>NumThreads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="NumThreadsValue">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="BatchsizeLayout">
          <item>
           <widget class="QLabel" name="BatchsizeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Batchsize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="BatchsizeValue">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="NumIterLayout">
          <item>
           <widget class="QLabel" name="NumIterLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of iterations to run for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>NumIter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="NumIterValue">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Lambda1Layout">
          <item>
           <widget class="QLabel" name="Lambda1Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Lambda1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Lambda1Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Lambda2Layout">
          <item>
           <widget class="QLabel" name="Lambda2Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Lambda2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Lambda2Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="PosAlphaLayout">
          <item>
           <widget class="QCheckBox" name="PosAlphaValue">
            <property name="text">
             <string>PosAlpha</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="PosDLayout">
          <item>
           <widget class="QCheckBox" name="PosDValue">
            <property name="text">
             <string>PosD</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="CleanLayout">
          <item>
           <widget class="QCheckBox" name="CleanValue">
            <property name="text">
             <string>Clean</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ModeLayout">
          <item>
           <widget class="QLabel" name="ModeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ModeValue">
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ModeDLayout">
          <item>
           <widget class="QLabel" name="ModeDLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ModeD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ModeDValue">
            <property name="maximum">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
